tests/JavaTest.java

```diff
@@ -1224,4 +1226,11 @@ static void TestVectorOfBytes() {
             System.exit(1);
         }
```

Comment:
This is really a separate unit test, I would extract to its own test.

Reply:
There are basically two tests happening within this one test fixture. Typically we would test a single thing per test so we can more readily find the issue if it breaks in the future. Especially since the second test is now commented out, I think it would be a good time to extract that to its own test and leave the TODO to fix in another PR.

Reply:
We appreciate the community contribution and want to ensure the quality of the code is constantly improved. The requested changes are simple refactorings and are at the same level I would ask for internally at Google. 

In terms of releases, there is really no rush to get things in to beat a release cut. If you look at the history of [releases](https://github.com/google/flatbuffers/releases) they are separated by 6 months+. We always advise to use `master` when feasible.

Reply:
@greenrobot FlatBuffers has had hundreds of contributors, and is being relied upon by a lot of people to have some level of quality.. we cannot get around having some level of code review to keep certain standards, I believe it benefits everyone. Most contributors appear to have no problems with this.

I guess it also depends on the topic. If this was a community video game I'm sure it be a lot of "fun". But we're in the business of keeping everyone's bits and bytes forwards and backwards compatibly accessible, which is by nature a pretty conservative business ;)

Reply:
@greenrobot FlexBuffers for java was introduced by me, a fellow contributor outside google as yourself. Since its quite new it is not bug-free, as you can tell. If you plan to use this code in your projects I would suggest keeping contributing as you have done so far. I am not sure you will get your fix shipped at a speed you probably need if you rely only on Google.

As for the release cadence, my understanding by chatting with @aardappel is that there is no technical reason to no ship the java artifacts more frequently. It is just a matter of setting up a better pipeline to publish the artifacts.  

Unfortunately, I have 0% free time for this whole month to look into that, but I guess that would be a great way to contribute, and probably will help to get the artifacts in the speed you need.

Feel free to join discord channel to discuss how we can speed up Java/Kotlin releases, in case you still want to contribute!